wire-ios/Wire-iOS/Sources/UserInterface/Settings/Backup/Adapters/MessageStoreAdapter/MessageStoreAdapter.swift

```diff
@@ -49,6 +49,139 @@ struct MessageStoreAdapter<MessageLocalStore>: MessageStoreProtocol, @unchecked
         }
     }
 
+    func addMessage(_ message: BackupMessageModel) async throws {
```

Comment:
This code is still a mess. Could we use existing code (`ConversationProtobufMessageProcessor`) to import models from `GenericMessage`?